com.ibm.wala.cast/src/main/java/com/ibm/wala/cast/loader/AstMethod.java

```diff
@@ -167,10 +167,6 @@ protected AstMethod(
     return cfg;
   }
 
-  public boolean hasCatchBlock() {
```

Comment:
Deleting since this method just calls itself in an infinite recursion. Clearly no other code has been calling it